added heart and trash icons

SachiGoto opened the pull request:
### Feature Description

Added heart icon to event modals on the map and event list page. 
Added trash icon to the favs list page.
Also, worked on responsiveness so it looks okay on mobile size.


![Screenshot 2023-11-01 at 6 00 24 PM](https://github.com/van-squad/map-t3/assets/70562492/7de7dfaa-be38-45ff-93a1-77687e321be4)
![Screenshot 2023-11-01 at 6 00 17 PM](https://github.com/van-squad/map-t3/assets/70562492/7f2b2d13-49af-4a78-919a-fa9e9fc7491e)
![Screenshot 2023-11-01 at 6 27 14 PM](https://github.com/van-squad/map-t3/assets/70562492/6e7820c5-b553-42d7-ab56-532e443f3cf4)


![Screenshot 2023-11-01 at 5 36 58 PM](https://github.com/van-squad/map-t3/assets/70562492/0388f28e-bb06-4b5f-8e3d-3224f0afe171)
![Screenshot 2023-11-01 at 5 36 43 PM](https://github.com/van-squad/map-t3/assets/70562492/56772a65-d51a-48e9-9a13-375aa8db4b9a)

![Screenshot 2023-11-01 at 5 37 13 PM](https://github.com/van-squad/map-t3/assets/70562492/1f7baf45-7a96-4f7f-bf5d-ef3369ce8a71)

![Screenshot 2023-11-01 at 6 27 00 PM](https://github.com/van-squad/map-t3/assets/70562492/dfef6b3f-bcd6-4e8d-9456-0ea76220b844)

### Related Issue
close #98 
<!--- Put issue number if it's related. ex) #8 -->
<!--- Delete it if not needed -->

### Something you want to mention

<!--- Make a note about some problems or questions -->
<!--- Delete it if not needed -->

### Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

SachiGoto committed:
added heart and trash icons

SachiGoto committed:
adjusted css

kei95 commented on src/app/(events)/eventlist/page.tsx:
```diff
@@ -25,6 +45,24 @@ const EventListPage = () => {
+
+   const { mutate } = trpc.favoriteEvents.addFavorite.useMutation();
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```
Don't do it 😤

SachiGoto replied:
Hi @kei95! It's nice to hear from you here haha. Thank you for reviewing my code! Would you mind providing more clarity on what you mean??

kei95 replied:
Haha didn't meat to be mean here but I love what you all have made so far! The comment was about the disable eslint comment. It's generally a good idea to avoid doing this or leave context of why you're doing this at really least

SachiGoto replied:
Got it! Thanks for hopping in and giving me advice!

keento0809 reviewed (approved):
@SachiGoto Thank you for modifying styling stuff a lot! LGTM, and I'm grateful if you can check what will happen when deleting `eslint-disable-next-line @typescript-eslint/no-unsafe-assignment` 🙏

SachiGoto committed:
removed diable eslint

SachiGoto commented:
> @SachiGoto Thank you for modifying styling stuff a lot! LGTM, and I'm grateful if you can check what will happen when deleting `eslint-disable-next-line @typescript-eslint/no-unsafe-assignment` 🙏

It worked without the line! I removed and updated my PR!

Lada496 reviewed (approved):
Thank you @SachiGoto!
